lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the REEF unit tests require a number of open files which is greater than the default open file limit on a number of Linux distributions such as Ubuntu 16.04/16.10.  This limit is controlled in the shell by the "ulimit -n" command
```

Comment:
Ok lets take a step back, why do we need a wiki and build instructions, seems redundant, I would vote for just having build instructions and remove the wiki to reduce having to maintain 2 different pieces, we can have a reference in the wiki to have the developers just read the readme file in each of the build directories and follow instructions there, what do you think?